vendor/mpdf/mpdf/src/Mpdf.php

```diff
@@ -2,41 +2,23 @@
 
 namespace Mpdf;
 
-use fpdi_pdf_parser;
-use pdf_parser;
-
 use Mpdf\Config\ConfigVariables;
 use Mpdf\Config\FontVariables;
 
-use Mpdf\Color\ColorConverter;
-use Mpdf\Color\ColorModeConverter;
-use Mpdf\Color\ColorSpaceRestrictor;
-
 use Mpdf\Conversion;
 
 use Mpdf\Css\Border;
 use Mpdf\Css\TextVars;
 
-use dokuwiki\plugin\dw2pdf\DokuImageProcessorDecorator as ImageProcessor;
```

Comment:
@Klap-in Thank you for pointing out the regression. This time a couple more changes were necessary. I'll try to add a unit test to prevent this from happening in future library upgrades.

Reply:
In the mpdf tests you could find examples of verifying the generated pdf. I think this test could show some relevant parts https://github.com/mpdf/mpdf/blob/development/tests/Mpdf/MpdfTest.php

Did you test manually if exporting uploaded images is working?